### PR TITLE
Add Cambridge ilab network to seed node

### PR DESCRIPTION
**Pull request by dougszumski**
Statically assigns an IP to the interface on the ilab network
and sets up the ilab network as the default gateway on the seed node.

### CODE REVIEW

File: etc/kayobe/networks.yml [hunk 1]
```diff
@@ -191,11 +194,20 @@ inspection_net_inspection_allocation_pool_end: "10.{{ inspection_net_vlan }}.102
 # inspection_net_mtu:
 # inspection_net_routes:
 
+# Cambridge ilab network IP information.
+ilab_net_vlan: 60
+ilab_net_cidr: "10.{{ ilab_net_vlan }}.0.0/16"
+ilab_net_routes:
+  # Default route onto the Cambridge ilab network
+  - cidr: "0.0.0.0/0"
+    gateway: "10.60.255.1"
```
Comment (JohnGarbutt) [inline etc/kayobe/networks.yml:203]: I wasn't 100% on this one, but I think this is correct.

**Review by JohnGarbutt — APPROVED** [review]

File: etc/kayobe/inventory/group_vars/seed/network-interfaces [hunk 2]
```diff
@@ -30,6 +30,9 @@ storage_mgmt_net_interface: "{{ provision_oc_net_interface }}.{{ storage_mgmt_ne
 # storage_mgmt_net_bridge_ports:
 # storage_mgmt_net_bond_slaves:
 
+# Admin backdoor to seed node
+ilab_net_interface: "{{ ilab_net_interface }}.{{ ilab_net_vlan }}"
```
Comment (JohnGarbutt) [inline etc/kayobe/inventory/group_vars/seed/network-interfaces]: should this be provision_oc_net_interface?

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]

File: etc/kayobe/inventory/group_vars/seed/network-interfaces (hunk 2 above)
Comment (dougszumski) [reply etc/kayobe/inventory/group_vars/seed/network-interfaces]: Ah, I see, so we use this network to provision the seed. I should have picked up on that..

Comment (dougszumski) [reply etc/kayobe/inventory/group_vars/seed/network-interfaces]: Good spot, avoided infinite recursion, thanks!

File: etc/kayobe/networks.yml (hunk 1 above)
Comment (dougszumski) [reply etc/kayobe/networks.yml:203]: For reference this is what we have on the seed at the moment:

[doug@cumulus-seed cumulus-kayobe-config]$ sudo ip route show
default via 10.60.255.1 dev em1.60

**Review by JohnGarbutt — APPROVED** [review]
Looks good now, thank you!

**Pull request merged**